tests/security/oscap_pci_dss_4/oscap_xccdf_remediate.pm

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'pci-dss-4' hardening in the 'scap-security-guide': mitigation mode
+# Maintainer: QE Security <none@suse.de>
+# Tags: poo#93886, poo#104943
```

Comment:
This Tags field stopped to make sense a lot of time ago, unless it is really specific for this, feel free to remove it, specially when more than one ticket is added is one start to accumulate and being less usuful.

Reply:
Fixed